Dockerfiles/Python/Dockerfile

```diff
@@ -40,11 +40,12 @@ RUN ./install-dapr-cli.sh
 RUN ./install-k3d-prerequisites.sh
 USER 4000
 RUN ./install-k3d-tooling.sh
-USER root
 
 WORKDIR /requirements
-
```

Comment:
please add the files directly from the template repo like this:
ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/requirements.txt /requirements